layers/meta-resin-intel/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -7,3 +7,15 @@ RESIN_CONFIGS_append_intel-corei7-64 = " nvme"
 RESIN_CONFIGS[nvme] = " \
     CONFIG_BLK_DEV_NVME=y \
     "
+
+#
+# Support for DLM module
+#
+RESIN_CONFIGS_append_intel-corei7-64 = " dlm"
+RESIN_CONFIGS_DEPS[dlm] = " \
+    CONFIG_SYSFS=y \
+    CONFIG_CONFIGFS_FS=y \
```

Comment:
I thought that dependencies on modules need to be built in. Now I see I am mistaken.